test/replication-luatest/quorum_misc_test.lua

```diff
@@ -1,7 +1,7 @@
 local t = require('luatest')
 local log = require('log')
 local fio = require('fio')
-local Cluster =  require('test.luatest_helpers.cluster')
+local Cluster =  require('luatest.replica_set')
```

Comment:
same as above, Cluster→cluster

anyway it's only used once on line 11

Reply:
Answered above.